Models/ViewModels.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Text.Json.Serialization;
+
+namespace LinguaNews.Models
+{
+    // It contains only the data we need to display on the page.
+    public class ArticleViewModel
+    {
+        public string Title { get; set; } = string.Empty;
+        public string Description { get; set; } = string.Empty;
+        public string Url { get; set; } = string.Empty;
+        public string UrlToImage { get; set; } = string.Empty;
+        public string SourceName { get; set; } = string.Empty;
+    }
+
+
+    // Add classes to deserialize the NewsData.io JSON response
+    // These match the fields in your ArticleData.cs but use System.Text.Json
```

Comment:
Avoid putting conversation in source code.  Put it in the Files Changed section.